packages/core/src/lib/select/index.ts

```diff
@@ -1,5 +1,5 @@
 export { SortOptions, type SortOption } from './search';
-export { default as SearchableSelect } from './searchable-select.svelte';
+export { default as TypeaheadInput } from './typeahead-input.svelte';
```

Comment:
**quibble (naming):** my brain is not cooperating with me when I try to read `typeahead`. I like ditching the "searchable select" nomenclature, but could we maybe name this `Combobox` instead to match ARIA?

I did a little digging in other Svelte component libraries and found that [skeleton](https://www.skeleton.dev/) ships a separate `Autocomplete` component that you can compose into a combobox. So if `Comboxbox` doesn't tickle you, I also like `AutocompleteInput` as a name